test/middleware_test.rb

```diff
@@ -0,0 +1,61 @@
+require "test_helper"
+
+class BustedTracerTest < Minitest::Test
+  class MockLogger
+    def initialize
+      @info = []
+    end
+    def info(info)
+      @info << info
+    end
+
+    def messages
+      @info
+    end
+  end
+
+  def test_returns_result_exactly
+    app = proc {|env| [200, {'Content-Type' => 'text/plain'}, ['OK']]}
+    middleware = BustedTracer.new(app)
+    assert_equal [200, {'Content-Type' => 'text/plain'}, ['OK']], middleware.call({})
+  end
+
+  def test_it_logs_cache_invalidations
+    log = MockLogger.new
+    app = proc {|env| [200, {'Content-Type' => 'text/plain'}, ['OK']]}
+    middleware = BustedTracer.new(app)
+    middleware.stub :logger, log do
+      middleware.call({})
+    end
+
+    assert_match(/methods=0 constants=0/, log.messages.last)
```

Comment:
The method's name is `test_it_logs_cache_invalidations`, but we aren't asserting on `"[Cache Invalidations]"`.  Why don't we add `"[Cache Invalidations]"` to this tests and maybe the others?